apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -151,6 +153,14 @@ export interface ICurrentVariantJson {
   variant: string | null; // Use `null` instead of `undefined` because `undefined` is not handled by JSON.
 }
 
+/**
+ * The paths used when resolving `pnpmStorePath`
+ */
+interface IPnpmOptionsRootPaths {
+  commonTempFolder: string;
+  projectRoot: string;
```

Comment:
We generally use the term `project` to refer to the individual projects inside the repo. I'd call this `repoRoot`

Reply:
Done.